builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -470,10 +470,11 @@ def updateSwaggerConfig() {
  * @param  repoName
  * @return
  */
-def cleanup(repoName) {
+def cleanup(serviceId) {
 
 	try {
-		scmModule.deleteProject(repoName)
+		def aclUrl = "${g_base_url}/jazz/acl/policies"
+		scmModule.updateAclPermission(serviceId, [], auth_token, aclUrl)
```

Comment:
Ideally, during the cleanup  - 
- Delete the project, call `deleteProject`
- Upon successful deletion (only), call `deletePolicies`

Reply:
Updated as per comments.